### DISCUSSION TITLE
Questions about Workbench layout

### DISCUSSION THREAD

**Discussion by juanfung** [datacarpentry/r-socialsci#457] [General]
@zkamvar I wasn't sure where to post this question/issue. I've gotten feedback that the "Overview" box on the lesson pages is a bit hard to read, as the text is broken up into two columns (Questions, Objectives) and the text wraps within each column. [Example](https://datacarpentry.org/r-socialsci/01-intro-to-r.html) from our lesson

I noticed it's not just our lesson, but a Workbench format ([Example from Workbench docs](https://carpentries.github.io/sandpaper-docs/episodes.html))

It looks a bit better when you collapse the menu on the right, but still not ideal (and collapsing shouldn't be necessary). I'm wondering if it would be possible to have two rows (rather than two columns) for Questions and Objectives?

**Comment by zkamvar** [comment]
That's a good question and it's been brought up before in https://github.com/carpentries/workbench/discussions/10#discussioncomment-2938302

I have an issue to work on this, but I won't be able to get to it until sometime in June https://github.com/carpentries/sandpaper/issues/299

**Comment by juanfung** [comment]
@zkamvar thank you and understand!
